cola/libdialect/routing.cpp

```diff
@@ -147,7 +147,7 @@ void RoutingAdapter::recordRoutes(bool refine) {
             vector<Point> refined_pts;
             size_t N = pts.size() - 2;  // the number of interior points
             // We expect that there are at least two route points.
-            COLA_ASSERT(N >= 0);
+            COLA_ASSERT((int) N >= 0);
```

Comment:
I'd rather have this:

            size_t M = pts.size();
            // We expect that there are at least two route points.
            COLA_ASSERT(M >= 2);
            // Get the number of interior points:
            size_t N = M - 2;